lib/grid.rb

```diff
@@ -0,0 +1,37 @@
+require_relative "./grid_creator"
+class Grid
+  include GridCreator
+  attr_reader :cells, :rows, :columns, :boxes, :unsolved_cells
+
+  def initialize(puzzle)
+    @cells = puzzle.split("").map { |v| Cell.new(v) }
```

Comment:
- Would it be an idea to call `#to_i` here so that your Cell is less dependent on the implementation of your grid?